### DISCUSSION TITLE
please add a support FLUX1-DEV-Q-NF4 , 6GB Model

### DISCUSSION THREAD

**Discussion by rakib91221** [lllyasviel/stable-diffusion-webui-forge#1085] [Q&A]
https://civitai.com/models/642967/flux1-dev-q-nf4?modelVersionId=719186
![11](https://github.com/user-attachments/assets/30e9c32b-81d5-4263-bbf0-29d7561cce34)

**Comment by dongxiat** [comment]
u should wait for GGUF model ... its super lightweight model
